docs/Release_Notes.txt

```diff
@@ -0,0 +1,497 @@
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+obsproc v1.1.0        -> released to NCO on Aug ??,2022
+                      -> implemented ?? ??,2022 (prior or in sync with GFSv16.3)
+
+
+List of all code/scripts modified with this release
+---------------------------------------------------
+M CMakeLists.txt
+R build-obsproc/*
+A cmake/FindMKL.cmake
+A cmake/FindNetCDF.cmake
+M docs/README.md
+  fix/CMakeLists.txt
+  fix/bufrtab.mods
+  fix/prepobs_anow.bufrtable
+  jobs/CMakeLists.txt
```

Comment:
Not sure what the purpose of listing every file in the release notes is. 
If anyone is interested in knowing which files have changed, a `git diff` can be performed.

Release notes should contain significant changes/upgrades in the product, bugfixes and any relevant information that the user needs to know e.g. API changes. 

What these jobs/scripts do belong in documentation, not release notes.

Reply:
will shorten the list of files to the ones with changes only